flight/Libraries/inc/uavtalk_priv.h

```diff
@@ -103,6 +103,8 @@ typedef struct {
 #define UAVTALK_TYPE_OBJ_ACK   (UAVTALK_TYPE_VER | 0x02)
 #define UAVTALK_TYPE_ACK       (UAVTALK_TYPE_VER | 0x03)
 #define UAVTALK_TYPE_NACK      (UAVTALK_TYPE_VER | 0x04)
+#define UAVTALK_TYPE_FILEREQ   (UAVTALK_TYPE_VER | 0x10)
```

Comment:
Assign new message types for file data is in the commit message? :p 

The big thing (not documented yet) is atomically shoving ~3 responses in every time there's a req to keep overhead down, and to be able to detect loss.